ceph.spec.in

```diff
@@ -874,6 +874,8 @@ CEPH_USER_ID="-u 167"
 %{_sbindir}/groupadd ceph $CEPH_GROUP_ID -o -r 2>/dev/null || :
 %{_sbindir}/useradd ceph $CEPH_USER_ID -o -r -g ceph -s /sbin/nologin -c "Ceph daemons" -d %{_localstatedir}/lib/ceph 2> /dev/null || :
 
+[-d /var/run/ceph ] && chown ceph:ceph /var/run/ceph
```

Comment:
There is a typo (missing space character) between [ and -d. Also, we don't need to do it this way, the permissions/user/group for /var/run/ceph are already being set in ceph package file list. (~line 816)